cli/tests/test_next_prev_commands.rs

```diff
@@ -265,10 +337,18 @@ fn test_prev_onto_root_fails() {
     test_env.jj_cmd_ok(&repo_path, &["commit", "-m", "second"]);
     test_env.jj_cmd_ok(&repo_path, &["commit", "-m", "third"]);
     test_env.jj_cmd_ok(&repo_path, &["commit", "-m", "fourth"]);
-    // The root commit is before "first".
-    let stderr = test_env.jj_cmd_failure(&repo_path, &["prev", "6"]);
+    insta::assert_snapshot!(get_log_output(&test_env, &repo_path), @r###"
+    @  mzvwutvlkqwt
+    ◉  zsuskulnrvyr fourth
+    ◉  kkmpptxzrspx third
+    ◉  rlvkpnrzqnoo second
+    ◉  qpvuntsmwlqt first
+    ◉  zzzzzzzzzzzz
+    "###);
+    // The root commit is before "first", which is 5 commits back.
+    let stderr = test_env.jj_cmd_failure(&repo_path, &["prev", "5"]);
```

Comment:
Perhaps, the test name is misleading. I think this test checks if "prev beyond root" fails.

Reply:
What about the error message? Do you agree that it could be misunderstood to mean that `@` has no ancestor 5 commits back? The fact that the test itself originally used 6 instead of 5 (`@` is 6 commits from the root) makes me think it's not an impossibility.

Reply:
My opinion on the error message is: it definitely improved for the root case, but no longer has a _why_ for the common case which loses clarity.

Reply:
:/ there's probably no perfect answer. I guess it ultimately depends on the user's understanding that the offset is relative to the parent(s) of the wc commit.

I obviously had a profound lack of understanding when it comes to that. 😂

Reply:
Ok, looks like we're keeping the error message for now so I'm going to close this thread.